content/about.html

```diff
@@ -135,44 +86,66 @@ <h2 class="no-link">
-                If you want to learn how to build on the web, or how to build on
-                the web better, you've come to the right place. A basic
-                knowledge of HTML/CSS/JS is all you need to get started using
-                the tools and guidance on web.dev to:
+                To get started, all you need is a basic understanding of HTML, CSS, and JavaScript.
               </p>
               <div class="whats-on__card-row">
                 <div class="whats-on__card">
                   <h3 class="no-link">Measure</h3>
                   <p>
-                    Using Lighthouse—an open-source, automated tool for running
-                    diagnostics on any web page—will help you understand how and
-                    where to start improving the quality of your web page. It has
-                    audits for performance, accessibility, progressive web apps, and
-                    more.
+                    Audit your site with <a href="https://developers.google.com/web/tools/lighthouse/" target="_blank">Lighthouse</a> 
+                    to get guidance you can use to improve:
                   </p>
+                  <ul>
+                    <li>Performance</li>
+                    <li>Progressive Web App status</li>
+                    <li>Accessibility</li>
+                    <li>Use of best practices</li>
+                    <li>Search engine optimization</li>
+                  </ul>
                   <a href="/measure"
                       class="button button-flat gc-analytics-event"
                       data-category="web.dev"
                       data-label="about, explore measure"
-                      data-action="click">Learn More</a>
+                      data-action="click">Test my site</a>
                 </div>
                 <div class="whats-on__card">
                   <h3 class="no-link">Learn</h3>
                   <p>
-                    Organized around the core principles of what makes a better web,
-                    you'll find a comprehensive collection of actionable guides,
-                    tutorials, and codelabs to help you understand how to make your
-                    site better, and even practice doing it.
+                    Practice new skills for the modern web and expand your knowledge on:
                   </p>
+                  <ul>
+                    <li>Load speed</li>
+                    <li>Network resilience</li>
+                    <li>Data security</li>
+                    <li>Site discoverability</li>
+                    <li>Accessibility for all users</li>
+                  </ul>
                   <a href="/learn"
                       class="button button-flat gc-analytics-event"
                       data-category="web.dev"
                       data-label="about, explore learn"
-                      data-action="click">Learn More</a>
+                      data-action="click">Keep learning</a>
```

Comment:
For consistency with my upcoming homepage changes, I'd like to change "Keep learning" to "Explore topics"